Prox/Prox/PlaceDetails/FilterViewController.swift

```diff
@@ -86,7 +95,7 @@ class FilterViewController: UIViewController {
 
         showFilterConstraints += [
             container.topAnchor.constraint(equalTo: view.topAnchor),
-            placeCountLabel.topAnchor.constraint(equalTo: topLayoutGuide.bottomAnchor, constant: spacing),
+            placeCountLabel.topAnchor.constraint(equalTo: topLayoutGuide.topAnchor, constant: 45),
```

Comment:
I don't  think this is necessary - we have `placeCount` centerY later. Maybe this was necessary because it references topLayoutGuide as the anchor, rather than the container view?

If it is, nit: `topLayoutGuide.topAnchor` -> `view.topAnchor` I think is preferred.